static/sass/styles.scss

```diff
@@ -313,3 +313,10 @@ sup {
     margin-bottom: 1rem;
   }
 }
+
+/// XXX Override for p-strip--image to hide the background at small viewports
+.u-no-background--small {
+  @media (max-width: $breakpoint-medium) {
+    background-image: none !important;
```

Comment:
Maybe. I don't think that I can apply it across the board. Some strips keep their background and would have text-visibility issues. This might be fixed as the "set the color, set the background color" rule is applied more in Vanilla and locally.